pkg/machinepool/machinepool.go

```diff
@@ -29,6 +33,49 @@ func NewMachinePoolService() MachinePoolService {
 	return &machinePool{}
 }
 
+// ListMachinePools lists all machinepools (or, nodepools if hypershift) in a cluster
+func (m *machinePool) ListMachinePools(r *rosa.Runtime, clusterKey string, cluster *cmv1.Cluster, isHypershift bool) {
+	// Load any existing machine pools for this cluster
+	r.Reporter.Debugf("Loading machine pools for cluster '%s'", clusterKey)
+	var err error
+	var machinePools []*cmv1.MachinePool
+	var nodePools []*cmv1.NodePool
+	if !isHypershift {
```

Comment:
NP : we could remove a lot of if/else if we break up this func. 

something like 
```suggestion
	var err error
	var output string
	if cluster.Hypershift().Enabled() {
		output, err = m.getNodePoolsString(.......)
	}
	output, err = m.getMachinePoolsString(.......)
	if err != nil {
		return err
	}

	// Create the writer that will be used to print the tabulated results:
	writer := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
	fmt.Fprint(writer, output)
	writer.Flush()
	return nil
```